jax/experimental/jax2tf/examples/tflite/mnist/README.md

```diff
@@ -0,0 +1,92 @@
+# MNIST classification
+
+This directory contains the code necessary to train a simple convolution network on the MNIST dataset, and subsequently convert it using jax2tf into a model that can be converted to tflite format.
+
+This example is based on the [TensorFlow Lite demo of building a handwritten digit classifier app with TensorFlow Lite](https://developer.android.com/codelabs/digit-classifier-tflite). The model code is taken from [FLAX Linen Mnist Example](https://github.com/google/flax/tree/master/linen_examples/mnist).
+
+## Requirements
+
+* TensorFlow dataset mnist will be downloaded and prepared automatically, if necessary
+
+## Training the model
+
+You can train and export the model to tflite format yourself running
+```bash
+$ python mnist.py
+```
+
+The training itself takes only about 1 minute, assuming the dataset has already been downloaded. The dataset will be downloaded directly into a data/ directory in the /tmp/jax2tf/mnist directory. The training loop runs for 3 epochs, and the model is then converted to TensorFLow Lite and subsequently saved into /tmp/jax2tf/mnist/mnist.tflite.
+
+### Convert to TF function using jax2tf converter
+Write a predict function from the trained model and use the jax2tf converter to convert it to a TF function.
+
+```
+def predict(image):
+   return CNN().apply({'params': optimizer.target}, image)
+```
+```
+# Convert Flax model to TF function.
+tf_predict = tf.function(
+    jax2tf.convert(predict, enable_xla=False),
+    input_signature=[
+        tf.TensorSpec(shape=[1, 28, 28, 1], dtype=tf.float32, name='input')
+    ],
+    autograph=False)
+```
+
+### Convert to TensorFlow Lite
+[TensorFlow Lite converter](https://www.tensorflow.org/lite/convert#python_api_) provides a number of ways to convert a TensorFlow model to the TensorFlow Lite format. Here we use the `from_concrete_functions` API.
+
+```
+# Convert TF function to TF Lite format.
+converter = tf.lite.TFLiteConverter.from_concrete_functions(
+    [tf_predict.get_concrete_function()])
+
+converter.target_spec.supported_ops = [
+    tf.lite.OpsSet.TFLITE_BUILTINS,  # enable TensorFlow Lite ops.
+    tf.lite.OpsSet.SELECT_TF_OPS  # enable TensorFlow ops.
+]
+tflite_float_model = converter.convert()
+```
+
+Note that tf.lite.OpsSet.SELECT_TF_OPS are required here to run regular TensorFlow ops (as compared to TensorFlow Lite ops) in the TensorFlow Lite runtime.
+
+### Apply quantization
+
+The converted format is no different than one converted from a regular TensorFlow Model. We can apply quantization as usual.
```

Comment:
Suggestions:
- I'd not use "as usual" for those who are new to on-device stuff - instead, I suggest to include a link to quantization if someone wants to study it
- Because the steps are sequential, I suggest to include "and now the next step/stage is...":

```suggestion
The next step is to perform [quantization](https://www.tensorflow.org/lite/performance/post_training_quantization). Because the converted format is no different than the one converted from a regular TensorFlow SavedModel, you can apply quantization:
```

LMKWYT @qiuminxu